docs/tutorials/sml-classification.ipynb

```diff
@@ -0,0 +1,818 @@
+    "- Regularization types: `onsite` (λ) for direct sparsity and `alpha` for a ratio-based trade-off between interaction and onsite terms\n",
+    "- Auto-regularization: set `regularization=\"auto\"` with a target selection ratio to adapt sparsity automatically\n",
+    "- Optimizer options: simulator versus QPU, repetitions, classical optimizer and its options, transpilation depth, and runtime sampler/estimator settings\n",
+    "\n",
+    "Benchmarks in the documentation show that accuracy improves as the number of learners (qubits) increases on challenging problems, with the quantum classifier matching or exceeding a comparable classical ensemble. In this tutorial, you will reproduce the workflow end-to-end and examine when increasing ensemble width or switching to adaptive regularization yields better F1 at reasonable resource usage. The result is a grounded view of how a quantum optimization step can complement, rather than replace, classical ensemble learning in real applications."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "55b94021",
+   "metadata": {},
+   "source": [
+    "## Requirements\n",
+    "\n",
+    "Before starting this tutorial, ensure you have the following packages installed in your Python environment:\n",
+    "\n",
+    "- `qiskit[visualization]~=2.1.0`\n",
+    "- `qiskit-serverless~=0.24.0`\n",
+    "- `qiskit-ibm-runtime v0.40.1`\n",
+    "- `qiskit-ibm-catalog~=0.8.0`\n",
+    "- `scikit-learn==1.5.2`\n",
+    "- `pandas>=2.0.0,<3.0.0`\n",
+    "- `imbalanced-learn~=0.12.3`"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "7db2e559",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "\n",
+    "In this section, we initialize the Qiskit Serverless client and load the Singularity Machine Learning – Classification function provided by Multiverse Computing.\n",
+    "With Qiskit Serverless, you can run hybrid quantum–classical workflows on IBM managed cloud infrastructure without worrying about resource management.\n",
+    "You will need an IBM Quantum Platform API key and your cloud resource name (CRN) to authenticate and access Qiskit Functions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d4fe7ee1-21ce-445c-b151-598cd4cf9227",
+   "metadata": {},
+   "source": [
+    "### Download the dataset\n",
+    "\n",
+    "To run this tutorial, we use a preprocessed **grid stability classification dataset** containing power system sensor readings labeled as *stable* or *unstable*.\n",
+    "The dataset is publicly hosted in the Multiverse Computing fork of the Qiskit documentation repository.\n",
```

Comment:
@sepehr-multiverse - your branch won't persist after we merge this PR, so this method won't work. I spoke to our engineers and they are ok with pointing to the files here in qiskit/documentation (the main branch). Can you please adjust this text to reflect downloading the data from there instead?